docs/install/any-kubernetes-cluster.md

```diff
@@ -385,6 +385,10 @@ At this point, you have a basic installation of Knative Serving!
 
 ### Optional Serving extensions
 
+You can apply extensions to customize your Knative Serving installation.
+For more information about the extensions you can apply, see
+[Installation files](./custom-install/custom-install.md#knative-serving-installation-files).
```

Comment:
Sorry, one more thing - these links need to be updated to the new filename 🙂

Reply:
Addressed!